docs/tutorials/cloudflare.md

```diff
@@ -6,34 +6,28 @@ Make sure to use **>=0.4.2** version of ExternalDNS for this tutorial.
 
 ## CloudFlare SDK Migration Status
 
-ExternalDNS is currently migrating from the legacy CloudFlare Go SDK v0 to the modern v4 SDK to improve performance, reliability, and access to newer CloudFlare features. The migration status is:
+ExternalDNS has been fully migrated from the legacy CloudFlare Go SDK v0 to the modern v5 SDK to improve performance, reliability, and access to newer CloudFlare features.
```

Comment:
```suggestion
Since v0.20.0, ExternalDNS has been fully migrated from the legacy CloudFlare Go SDK v0 to the modern v5 SDK to improve performance, reliability, and access to newer CloudFlare features.
```